CMakeLists.txt

```diff
@@ -154,17 +154,19 @@ endif ()
 # Make sure the final executable has symbols exported
 set (CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -rdynamic")
 
-find_program (OBJCOPY_PATH NAMES "llvm-objcopy" "llvm-objcopy-11" "llvm-objcopy-10" "llvm-objcopy-9" "llvm-objcopy-8" "objcopy")
-if (OBJCOPY_PATH)
-    message(STATUS "Using objcopy: ${OBJCOPY_PATH}.")
-
-    if (ARCH_AMD64)
-        set(OBJCOPY_ARCH_OPTIONS -O elf64-x86-64 -B i386)
-    elseif (ARCH_AARCH64)
-        set(OBJCOPY_ARCH_OPTIONS -O elf64-aarch64 -B aarch64)
+if (OS_LINUX)
```

Comment:
FreeBSD?

Reply:
`OBJCOPY_PATH` is used only under `if (OS_LINUX)` right now.

Reply:
Ok.